libs/blocks/global-navigation/global-navigation.js

```diff
@@ -315,7 +315,7 @@ class Gnav {
         window.addEventListener('onImsLibInstance', () => this.imsReady());
         return;
       }
-      lanaLog({ message: 'GNAV: Error with IMS', e, tags: 'errorType=info,module=gnav' });
+      lanaLog({ message: 'Error with IMS', e, tags: 'errorType=info,module=gnav' });
```

Comment:
Unrelated, but I'd personally only change strictly unit test related things in a _unit test_ PR. Makes it easier to merge & u don't want things to accidently break prod in a PR that was supposed to just changes tests.

(It's definitely fine to keep it)

Reply:
Valid point 👍 It's better to keep the purpose of the PR clean, made the change 👍